Assets/Scripts/Augment/GunFactory.cs

```diff
@@ -131,9 +132,18 @@ public void InitializeGun()
 
         modifiers.OrderByDescending(modifier => (int) modifier.GetPriority()).ToList();
         modifiers.ForEach(modifier => modifier.ModifyProjectile(ref projectileController));
-
         gunController.onInitialize?.Invoke(gunController.stats);
     }
 
+    // Lagrer våpen kombo til spiller
```

Comment:
Maybe add a // TODO: Implement save/get instead og norwegian comments

Reply:
Comments and pseudocode has been removed.